PC/svm/svm_train_and_save/svm_train_and_save.cpp

```diff
@@ -23,14 +34,27 @@ int main()
 
     // Set up training data
     std::vector<float> labels;
-            
-    // Read training label
+    std::vector<float> training_data;
+
     for (auto element : training_data_json)
     {
+        // Read training data
+        std::vector<float> training_element = svm_build_single_element(element);
```

Comment:
Do we want to do it manually every time?
Why not prepare_data_mat(training_data_json); ?

Reply:
You are right, will prepare